cubeviz/data_factories/__init__.py

```diff
@@ -2,3 +2,287 @@
 from .jwst import read_jwst_data_cube
 from .kmos import read_kmos_data_cube
 from .manga import read_manga_data_cube
+from glue.core import Data
+from glue.core.coordinates import coordinates_from_header
+from ..listener import CUBEVIZ_LAYOUT
+from ..layout import FLUX, ERROR, MASK
+
+from os.path import basename, splitext
+import yaml
+import os
+import glob
+import logging
+from astropy.io import fits
+import numpy as np
+from glue.config import data_factory
```

Comment:
A bit nit-picky, but [according to PEP8](https://www.python.org/dev/peps/pep-0008/#imports):
> Imports should be grouped in the following order:
> * standard library imports
> * related third party imports
> * local application/library specific imports
> 
> You should put a blank line between each group of imports.

Reply:
switched around